version.sbt

```diff
@@ -1 +1 @@
-ThisBuild / version := "1.9.3-SNAPSHOT"
+ThisBuild / version := "2.0.0"
```

Comment:
Ah this should be "2.0.0-SNAPSHOT".

An interesting use case i felt. According to `gha-release`, the process should scan the changes and compare with base branch and should able to bump major version if it finds so. but as @emdash-ie mentioned it is not. I think just to double check we can confirm with @rtyley if possible

Reply:
> Ah this should be "2.0.0-SNAPSHOT".

Ah yes, I didn’t think of that.

> An interesting use case i felt. According to `gha-release`, the process should scan the changes and compare with base branch and should able to bump major version if it finds so. but as @emdash-ie mentioned it is not. I think just to double check we can confirm with @rtyley if possible

Agreed, it would be nice to understand why the release process disagrees with me about the need for a major version bump.

Reply:
I have updated this to add `-SNAPSHOT` now.

Reply:
Spoke with Roberto, he agrees something’s not right here – the version check doesn’t appear to be working on this repository. I think we’ll go ahead with this major-version bump in the meantime though, and look into that separately.